backend/app/domains/courses/crud.py

```
@@ -1,21 +1,32 @@
 from typing import List
 
+from fastapi import HTTPException
 from fastapi_pagination import paginate
 from fastapi_pagination.ext.sqlmodel import paginate
```

Ruff (0.11.9)

Redefinition of unused `paginate` from line 4

(F811)